api/src/test/java/org/openmrs/OpenmrsTestsTest.java

```diff
@@ -9,7 +9,8 @@
  */
 package org.openmrs;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
@achilep will you also add a Test? I believe thats a class testing whether an OpenMRS test is properly setup.

Reply:
@teleivo there I don't understand well .

> will you also add a Test?

Reply:
@teleivo surely I plan to increase the coverage of the tests. I plan to do it at the end of the migration.

Reply:
This file is like a metatest, going through all test classes and checking they have a test, ...
Can you please add another test class like ModuleFileParserUnitTest to this commit for example that is a unit test

Reply:
Sorry, I have to correct myself. ModuleFileParserUnitTest extends BaseContextMockTest so we should not include this one in the commit. So any other Test class that does not extend any of these `BaseContext` classes. You did already migrate a few previously.

Reply:
thanks @teleivo . done